examples/python/streaming.py

```diff
@@ -0,0 +1,76 @@
+from pyspark import SparkContext
+from pyspark.sql import Column, DataFrame, SparkSession, SQLContext, functions
+from pyspark.sql.functions import *
+from py4j.java_collections import MapConverter
+from delta.tables import *
+import shutil
+import random
+import threading
+
+# Clear previous run delta-tables
+files = ["/tmp/delta-table", "/tmp/delta-table2", "/tmp/delta-table3"]
+for i in files:
+    try:
+        shutil.rmtree(i)
+    except:
+        pass
+
+# Create SparkContext
+sc = SparkContext()
+sqlContext = SQLContext(sc)
+
+spark = SparkSession \
+    .builder \
+    .appName("streaming") \
+    .master("local[*]") \
+    .getOrCreate()
+
+# Create a table(key, value) of some data
+data = spark.range(8)
+data = data.withColumn("value", data.id + random.randint(0, 5000))
+data.write.format("delta").save("/tmp/delta-table")
+
+# Stream writes to the table
+print("####### Streaming write ######")
+streamingDf = spark.readStream.format("rate").load()
+stream = streamingDf.selectExpr("value as id").writeStream.format("delta").option("checkpointLocation", "/tmp/checkpoint").start("/tmp/delta-table2")
```

Comment:
break this into smaller lines.

streamingDf.selectExpr("value as id").writeStream
   .format("delta")
   .option("checkpointLocation", "/tmp/checkpoint")
   .start("/tmp/delta-table2")